### PR TITLE
IA-3040 fix bug non default dataproc cluster on PPW projects

### PR DESCRIPTION
**Pull request by Qi77Qi**
https://broadworkbench.atlassian.net/browse/IA-3040

Use the same IP range as [RBS](https://github.com/DataBiosphere/terra-resource-buffer/blob/master/src/main/java/bio/terra/buffer/service/resource/flight/CreateSubnetsStep.java#L53)

`northamerica-northeast2` and `europe-central2` don't exist in RBS yet, pending https://github.com/DataBiosphere/terra-resource-buffer/pull/190.

This might mean that things are probably broken for these 2 regions until RBS supports them since we use subnet created by RBS if certain labels exist.

Verify things are boken today:
* Create a Dataproc cluster in non default region, and run hail tutorial up to `users['occupation'].describe()` in https://hail.is/docs/0.2/tutorials/03-tables.html, and go to yarn page, and find the worker's log, you'll see sth like 
```
2021-11-10 21:01:25,124 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: saturn-19cb0407-8b0b-4377-b84b-b14c28e2e9e1-m/10.154.0.2:8031. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
```

Tested out the same thing with this branch in fiab and no longer seeing the error

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by Qi77Qi** [comment]
fiab issue...jenkins retest

File: http/src/main/resources/reference.conf [hunk 1]
```diff
@@ -231,32 +231,32 @@ vpc {
...
+        northamerica-northeast1 = ["10.162.0.0/20"]
+        northamerica-northeast2 = ["10.188.0.0/20"]
+        southamerica-east1 = ["10.158.0.0/20"]
+        us-central1 = ["10.128.0.0/20"]
+        us-east1 = ["10.142.0.0/20"]
+        us-east4 = ["10.150.0.0/20"]
+        us-west1 = ["10.138.0.0/20"]
+        us-west2 = ["10.168.0.0/20"]
+        us-west3 = ["10.180.0.0/20"]
+        us-west4 = ["10.182.0.0/20"]
```
Comment (kyuksel) [inline http/src/main/resources/reference.conf:261]: Can we add a comment telling where these values come from?

**Review by yonghaoy — APPROVED** [review]

File: http/src/main/resources/reference.conf (hunk 1 above)
Comment (yonghaoy) [reply http/src/main/resources/reference.conf:261]: https://cloud.google.com/vpc/docs/vpc#ip-ranges

**Review by kyuksel — APPROVED** [review]
👍🏼 pending testing in different locations

**Comment by Qi77Qi** [comment]
going to merge this PR as is...I was able to verify things won't work in dataproc today in a non-default region via running a few cmds in hail tutorial to generate traffic, and then looking at spark UI, go to worker's log, see the following message 
```
2021-11-10 19:22:03,529 INFO org.apache.hadoop.ipc.Client: Retrying connect to server: saturn-19cb0407-8b0b-4377-b84b-b14c28e2e9e1-m/10.154.0.2:8031. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
```

This would be pretty hard to test in auto-tests, and this code will likely be removed with new RBS projects (for existing projects, there's already firewall created)

**Pull request merged**